src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -73,17 +72,19 @@ class LedgerImpl(vm: VM, blockchainConfig: BlockchainConfig) extends Ledger with
     *
     * @param block
     * @param blockchain
-    * @param storages
     * @param signedTransactionValidator
     */
   private[ledger] def executeBlockTransactions(
     block: Block,
-    blockchain: Blockchain,
-    storages: BlockchainStorages,
-    signedTransactionValidator: SignedTransactionValidator):
+    blockchain: BlockchainImpl,
+    signedTransactionValidator: SignedTransactionValidator,
+    readOnly: Boolean = false):
```

Comment:
I can only see `executeBlockTransactions` called from one place.... so readonly is *always* false ?